src/services/FlawService.ts

```diff
@@ -42,10 +42,9 @@ export async function beforeFetch(options: OsidbFetchOptions) {
       return;
     }
     try {
-      const flaw = await getFlawUpdatedDt(flawId);
-      options.data.updated_dt = flaw.updated_dt;
-      if (!flaw.updated_dt) {
-        console.error('During multi-stage operation, an updated_dt could not be fetched', flaw, flawId);
+      const updated_dt = await getUpdatedDt(options.url);
```

Comment:
Since we no longer fetch only Flaw but the model fetched here might be various (Affect, Reference, etc.) we should change wording of the error bellow (lines 51,52)

```
console.error('Problem fetching flaw for sequential update:', error);
throw new Error('Problem fetching flaw for sequential update');
```